docs/server/guides/software/picking.md

```diff
@@ -1,26 +1,31 @@
+Now you need to think about what features your server needs, like chests, combat, redstone etc. If your server doesn't need many vanilla features you may want to look into using a server software built from the ground up, like [Minestom](https://minestom.net) or [Krypton](https://kryptonmc.org). The vanilla server will work for whatever you decide but will not have the performance compared to software without all the features.
 
-### For MC 1.13+
+### Vanilla Software
 
-A good server jar to use is [Purpur](https://github.com/pl3xgaming/Purpur), it optimizes performance a lot and adds a lot of features (FLYING SQUIDS :D) to help optimize even further or just some fun cool things to do.
+If you need the vanilla software's features, trying to decide between a server software for a vanilla server is hard, there are many forks claiming to be the best. If you don't need any plugins on your server you can use a [Fabric](https://fabricmc.net) server with server-side optimization mods. The most commonly used vanilla software with plugin support is [SpigotMC](https://www.spigotmc.org) and its forks, [PaperMC](https://papermc.io), [Purpur](https://purpur.pl3x.net) etc. Another popular vanilla software with plugin support is [SpongeVanilla](https://www.spongepowered.org/downloads/spongevanilla), its primary use is in forge-spigot hybrid servers, in the form of [SpongeForge](https://www.spongepowered.org/downloads/spongeforge), but it can be used with just vanilla. The [SpongeAPI](https://www.spongepowered.org/downloads/spongeapi) doesn't have as big of a variety of plugins created for it as [SpigotMC](https://www.spigotmc.org) so you might need to create some of the plugins yourself. There is server software that is designed/optimized for faction/KitPVP or some other types of gamemodes but many are mass-produced and are abandoned quickly.
 
-### For MC 1.12 and below
+### Custom Software
 
-[PaperMC](https://papermc.io) is generally the best fork for these versions and not much more is needed.
+Custom software is created for better performance mainly targetting minigame servers. Some are trying to recreate the vanilla server with better performance and others are just an API and you need to recode every feature you want in the game. These are mostly good for minigame servers as the vanilla recodes are likely not finished.
 
-**JUST PLEASE DO NOT USE THE VANILLA JAR!**
+### Modded Software
 
-## [Optimization](optimization.md)
+If you want to play with mods you will need a modded server software. There are a few of these but the most popular/still updated ones are [Fabric](https://fabricmc.net) and [Forge](https://files.minecraftforge.net/net/minecraftforge/forge). There are mod versions of plugins but not as much of a wide variety, hybrid servers are better for this which will be talked about later in this article. [Fabric](https://fabricmc.net) is more commonly used for servers with mods that act as plugins.
```

Comment:
> hybrid servers are better for this which will be talked about later in this article.

The next paragraph is this, so maybe "hybrid servers (explained below) are better for this"